buildspec.yml

```diff
@@ -5,8 +5,11 @@ phases:
       java: openjdk8
       python: 3.7
     commands:
+      -  pip install --upgrade 'pyyaml < 6.0'
       -  pip install --upgrade 'six==1.15.0'
-      -  pip install --upgrade 'pyyaml>=5.4'
+      -  pip install boto3 --upgrade
+      -  pip install --upgrade 'docker~=4.0'
```

Comment:
Why is the `docker` dependency needed?

Reply:
This dependency already exists. We just had to change the version to fix the build issue.